docs/resources/extra_disk_size.md

```diff
@@ -46,10 +72,156 @@ data "cloudamqp_nodes" "nodes" {
 }
 ```
 
+</details>
+
+<details>
+  <summary>
+    <b>
+      <i>AWS extra disk size without downtime</i>
+    </b>
+  </summary>
+
+```hcl
+# Configure CloudAMQP provider
+provider "cloudamqp" {
+  apikey = var.cloudamqp_customer_api_key
+}
+
+# Instance
+resource "cloudamqp_instance" "instance" {
+  name   = "Instance"
+  plan   = "bunny-1"
+  region = "amazon-web-services::us-west-2"
+  rmq_version = "3.10.1"
```

Comment:
Omit `rmq_version` from all examples? As it is optional and latest will be used (less documentation that will go stale)

Reply:
Yes been thinking similar thoughts. Planned to do some spring cleaning in the documentation and logging later on. Also have a bunch of good examples that can be added used for issues, PR and overall testing.